Community-Supported/pandas-to-hyper/README.md

```diff
@@ -0,0 +1,76 @@
+
+# hyper-from-dataframe
+## Create a Hyper File from a Pandas DataFrame
+
+![Community Supported](https://img.shields.io/badge/Support%20Level-Community%20Supported-53bd92.svg)
+
+This Python script demonstrates how to create a `.hyper` file (Tableau's Hyper database format) from a pandas DataFrame. It uses Tableau's Hyper API to define a table structure, insert the data from the DataFrame, and save it as a `.hyper` file.
+
+## Get Started
+
+### Prerequisites
+
+Before running the script, ensure you have the following installed:
+
+- Python >= 3.6
+- The required dependencies listed in `requirements.txt`.
+
+### Install Dependencies
+
+To install the necessary dependencies, run the following command:
+
+```bash
+pip install -r requirements.txt
+```
+
+### Running the Script
+
+To run the script and generate the `.hyper` file, execute:
+
+```bash
+python hyper_from_dataframe.py
```

Comment:
wrong file name

Reply:
Good catch, I pushed an updated README with the correct filename to run the script.